generate/config.go

```diff
@@ -153,6 +156,23 @@ func (c *Config) ValidateAndFillDefaults(baseDir string) error {
 		c.ExportOperations = pathJoin(baseDir, c.ExportOperations)
 	}
 
+	// Calculate the fully qualified package path
+	genPkgPath, err := filepath.Rel(c.baseDir, c.Generated)
+	if err != nil {
+		return fmt.Errorf("finding relative path to generated package: %w", err)
+	}
+	pkgs, err := packages.Load(&packages.Config{
+		Dir:  c.baseDir,
+		Mode: packages.NeedName | packages.NeedModule,
+	}, "./"+filepath.Dir(genPkgPath))
+	if err != nil {
+		return fmt.Errorf("loading generated package: %w", err)
+	}
```

Comment:
> would this break things if the directory where you're putting the output is empty

Yes, it would break, but that would be the same as trying to bind to an empty package, right?

> does this require that baseDir is a valid Go package?

It shouldn't: it's just used as a base for the package search.